src/main/java/org/example/CoffeeMachine.java

```diff
@@ -0,0 +1,247 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import static jdk.internal.org.jline.utils.Colors.s;
+import static jdk.internal.org.jline.utils.Status.getStatus;
+import static org.example.ListMenuButton.*;
+import static java.lang.System.in;
+import static java.lang.System.out;
+import static org.example.DrinkType.CAPPUCCINO;
+import static org.example.DrinkType.ESPRESSO;
+
+public class CoffeeMachine {
+    private static final BufferedReader reader = new BufferedReader(new InputStreamReader(in));
+
+    private int waterLevel;
+    private int milkLevel;
+    private int coffeBeans;
+    private boolean isOn;
+
+    public boolean getIsOn() {
+        return isOn;
+    }
+
+    public void setIsOn(boolean isOn) {
+        this.isOn = isOn;
+    }
+
+    private final int maxWaterLevel = 100;
+    private final int maxMilkLevel = 50;
+    private final int maxCoffebeans = 20;
+
+    private int drinksPrepared;
+    private static final int CLEANING_THRESHOLD = 10;
+
+    public CoffeeMachine(int waterLevel, int milkLevel, int coffeBeans) {
+        this.waterLevel = waterLevel;
+        this.milkLevel = milkLevel;
+        this.coffeBeans = coffeBeans;
+        drinksPrepared = 0;
+
+    }
```

Comment:
public CoffeeMachine(int waterLevel, int milkLevel, int coffeBeans, int drinksPrepared) {
        this.waterLevel = waterLevel;
        this.milkLevel = milkLevel;
        this.coffeBeans = coffeBeans;
        this.drinksPrepared = 0;
    }